### PR TITLE
Make use of `import type`

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

### CODE REVIEW

**Comment by Page-** [comment]
@balena-ci retest

File: src/sbvr-api/permissions.ts [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-import {
+import type {
 	AbstractSqlModel,
 	AbstractSqlType,
 	AliasNode,
 	Relationship,
 	RelationshipMapping,
 	SelectNode,
 } from '@resin/abstract-sql-compiler';
+import type * as express from 'express';
```
Comment (dfunckt) [inline src/sbvr-api/permissions.ts:9]:
```suggestion
import type * as express from 'Express';
```

like elsewhere.

File: src/sbvr-api/sbvr-utils.ts [hunk 2]
```diff
@@ -57,6 +59,7 @@ import {
 	statusCodeToError,
 	TranslationError,
 	UnauthorizedError,
+	AnyObject,
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts:62]: Doesn't need to happen with with PR, but feels like `AnyObject` should be in `common_types.ts`, definitely not in `errors.ts`.

File: src/sbvr-api/sbvr-utils.ts [hunk 3]
```diff
@@ -73,20 +76,17 @@ import * as controlFlow from './control-flow';
...
 import { version as LF2AbstractSQLVersion } from '@resin/lf-to-abstract-sql/package.json';
 import { version as sbvrTypesVersion } from '@resin/lf-to-abstract-sql/package.json';
-import { Model } from '../config-loader/config-loader';
 import {
 	compileRequest,
 	getAndCheckBindValues,
 	isRuleAffected,
 } from './abstract-sql';
-import { OptionalField, RequiredField } from './common-types';
-import { AnyObject } from './sbvr-utils';
```
Comment (dfunckt) [inline src/sbvr-api/sbvr-utils.ts:89]: Ugh :/

**Pull request merged**